erpnext/accounts/doctype/payment_request/test_payment_request.py

```
@@ -851,3 +852,1027 @@ def test_payment_request_on_unreconcile(self):
+		customer = frappe.get_doc("Customer", "_Test Customer")
+
+		# Set a nonexistent address name
+		original_address = customer.customer_primary_address
+		customer.db_set("customer_primary_address", "NonExistentAddress12345", update_modified=False)
+
+		try:
+			pr = frappe.new_doc("Payment Request")
+			pr.party_type = "Customer"
+			pr.party = customer.name
+
+			contact, address = pr._get_party_contact_and_address()
```

Ruff (0.14.13)

Unpacked variable `contact` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)